### ISSUE TITLE
Keys are not synchronized in a cluster on import

### ISSUE THREAD

**Issue by yuce** [pilosa/pilosa#1784]
### Expected behavior

After importing `RowKeyColumnKey` type data, the keys should be queryable on all nodes in the cluster.

### Actual behavior

The keys seem to be not synchronized in the cluster even for smallish data. For 100 row keys and 100 column keys (10_000 bits), the keys seem to be synchronized. But for 1_000 row keys and  1_000 column keys (1_000_000 bits), keys only exist on the node which received the import. Other node(s) return `{"error":"executing: pilosa: translate store could not find or create key, translate store read only"}` when one of the row keys is queried.

### Steps to reproduce the behavior

Download and extract [cluster_cp.tar.gz](https://github.com/pilosa/pilosa/files/2672850/cluster_cp.tar.gz) which contains the configuration and data.

1. Run the first node: `./pilosa server -c node1.config.toml`
2. Run the second node: `./pilosa server -c node2.config.toml`
3. Import the `RowKeyColumnKey` data: `./pilosa import --host :10101 -e -i i1 -f f1 --index-keys --field-keys -s 100000 rk1000_ck1000_r.csv`
4. Query the data on the first node: ` curl localhost:10101/index/i1/query -d 'Count(Row(f1="1bfdabc4-0162-4d70-b6a6-75a45812c480"))'
{"results":[1000]}`
5. Expected result `{"results":[1000]}` is returned.
6. Query the data on the second node: `curl localhost:10102/index/i1/query -d 'Count(Row(f1="1bfdabc4-0162-4d70-b6a6-75a45812c480"))'`
7. Unexpected error: `{"error":"executing: pilosa: translate store could not find or create key, translate store read only"}` is returned.

There's the `create_RowKeyColumnKey_data.py` utility in the [cluster_cp.tar.gz](https://github.com/pilosa/pilosa/files/2672850/cluster_cp.tar.gz) to create data with different number of row/column keys.

### Information about your environment (OS/architecture, CPU, RAM, cluster/solo, configuration, etc.)

* Pilosa compiled from latest master: `17ea3f7f9403f5ca4a8ea0b4bdfccfefea31f5aa`
* Ubuntu 18.04

**Comment by travisturner** [comment]
@yuce I pushed a WIP PR with a potential fix for this.
Let me know if you think it addresses your issue. If so I'll continue with the TODOs in that PR.

**Comment by yuce** [comment]
I've run more tests, and it seems there are 3 outcomes as far as I tested:
* Second node returns `{"error":"executing: index not found"}`
* Bot the first and the second node returns `{"results":[1000]}` (expected outcome)
* Both the first and the second node returns `{"results":[0]}`.

**Comment by yuce** [comment]
`{"error":"executing: index not found"}` is resolved after waiting for some time. Trying to get find a consistent way of producing `{"results":[0]}`

**Comment by travisturner** [comment]
no, my last statement was incorrect. i haven't been able to reproduce that after all.

**Comment by yuce** [comment]
@travisturner There are no problems when I follow the exact steps, but I was running the second node after the import. In that case sometimes I receive the `index not found` error when I query the second node, but that is resolved after I run the second node and wait for a bit (should be due to synchronization). For 1000x1000 `RowKeyColumnKey` type data, the expected result is retrieved from the second after 20 seconds.

Reproducing `{"results":[0]}` case is the same:
1. Run the first node
2. Import the data
3. Run the second node
4. Immediately run a query on the second node

The outcome of the 4th step is usually the `index not found` error (resolved after some time) OR both node 1 and node 2 returns `{"results":[0]}`
